docs/checkov/task.md

```
@@ -0,0 +1,16 @@
+# Address Checkov Security Audit
+
+- [x] Analyze Checkov Results and Create Implementation Plan <!-- id: 0 -->
+- [x] Update `terraform/resources/alb/alb.tf` <!-- id: 1 -->
+    - [x] Enable `drop_invalid_header_fields`
```

markdownlint-cli2 (0.20.0)

[warning] Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)